tests/testthat/helper-mock-cli.R

```diff
@@ -0,0 +1,22 @@
+real_wcr <- wsl_compatible_run
+
+with_mocked_cli <- function(code, compile_ret, info_ret){
```

Comment:
@jgabry do we use `=` or `<-` for assignment in cmdstanr?

Reply:
`<-` (although I've found some `=` at times in the past, but I try to change them when I find them)